app/src/main/java/com/kamron/pogoiv/PokeInfoCalculator.java

```diff
@@ -206,6 +209,48 @@ public CPRange getCpRangeAtLevel(Pokemon pokemon, int lowAttack, int lowDefense,
     }
 
     /**
+     * Get the combined cost for evolving all steps between two pokemon, for example the cost from caterpie -> metapod is 12,
+     * caterpie -> butterfly is 12+25 = 27
+     * @param start which pokemon to start from
+     * @param end the end evolution
+     * @return the combined candy cost for all required evolutions
+     */
+    public int getCandyCostForEvolution(Pokemon start, Pokemon end){
+        Pokemon devolution = get(end.devoNumber);
+        Pokemon dedevolution = null;
+        if (devolution != null){ //devolution must exist for there to be a devolution of the devolution
+            dedevolution =  get(devolution.devoNumber);
```

Comment:
@harkin this is the potential call of `get(-1)`, but I agree I prefer the error-checking for `-1` here.